features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/messagecomposer/MessageComposerPresenter.kt

```diff
@@ -83,6 +83,11 @@ class MessageComposerPresenter @Inject constructor(
             canShareLocation.value = featureFlagService.isFeatureEnabled(FeatureFlags.LocationSharing)
         }
 
+        val canCreatePoll = remember { mutableStateOf(false) }
+        LaunchedEffect(Unit) {
+            canCreatePoll.value = featureFlagService.isFeatureEnabled(FeatureFlags.Polls)
```

Comment:
Maybe update the string here:
https://github.com/vector-im/element-x-android/blob/41d0d21c808353f91a0d4ceedcb4321a94cf3f31/libraries/featureflag/api/src/main/kotlin/io/element/android/libraries/featureflag/api/FeatureFlags.kt#L32

`"Render poll events in the timeline"` 

since it also a flag to create poll. I propose `"Create poll and render poll events in the timeline"`, but you choose.